docs/infrastructure/eks-quickstart.md

```diff
@@ -97,6 +97,41 @@ Create a cluster that includes all the required network and node setup:
     kubectl get pods -A
     ```
 
+## 4. Prepare to Setup the Ingress Controller
+
+Start by performing the following steps:
```

Comment:
Give a bit more context about why this is needed now (and how the setup is finished later on):

```suggestion
Prepare your infrastructure for the Ingress Controller setup, which is performed later during the installation process:
```